packages/cells/src/widget.ts

```diff
@@ -1272,20 +1273,21 @@ export abstract class AttachmentsCell extends Cell {
           CONTENTS_MIME_RICH
         ) as DirListing.IContentsThunk;
         if (model.type === 'file') {
-          this.updateCellSourceWithAttachment(model.name);
+          const URI = this._generateURI(model.name);
+          this.updateCellSourceWithAttachment(model.name, URI);
```

Comment:
There's already some mimetype filtering happening in this function when if makes the array of `supportedMimetypes`
https://github.com/jupyterlab/jupyterlab/blob/597742161fdb784bf4de7ef38f7c9d8b88b2d954/packages/cells/src/widget.ts#L1245-L1247

I hoped that this was doing enough filtering. I wasn't able to test this though, because I don't seem to be able to trigger this event. Should this be triggerable by dragging an image from the jupyterlab filebrowser?